go-controller/pkg/ovn/ovn.go

```diff
@@ -694,39 +695,37 @@ func (oc *Controller) GetNetworkPolicyACLLogging(ns string) *ACLLoggingLevels {
 	return &nsInfo.aclLogging
 }
 
-// Verify if controller can support ACL logging and validate annotation
-func (oc *Controller) aclLoggingCanEnable(annotation string, nsInfo *namespaceInfo) bool {
-	if !oc.aclLoggingEnabled || annotation == "" {
-		nsInfo.aclLogging.Deny = ""
-		nsInfo.aclLogging.Allow = ""
-		return false
-	}
+// aclLoggingUpdateNsInfo parses the provided annotation values and sets nsInfo.aclLogging.Deny and
+// nsInfo.aclLogging.Allow.
+// Parse the provided annotation string. Valid values for "allow" and "deny" are  "alert", "warning", "notice", "info",
+// "debug" and "".
+// In the following special cases, nsInfo.aclLogging.Deny and nsInfo.aclLogging.Allow. will both be reset to "", meaning
+// that logging will be switched off:
+// i) oc.aclLoggingEnabled == false
+// ii) annotation == ""
+// iii) annotation == "{}"
+// Also, if one of "allow" or "deny" can be parsed, but the other key is not present in the annotation, then assume
+// that this key should be disabled by setting its nsInfo value to "".
```

Comment:
I guess we need to add that the same will happen for invalid values. And probably test this too if you are interested.

Reply:
done